data/drak/indigenous.txt

```diff
@@ -287,7 +287,7 @@ ship "Embersylph"
 		"hull repair rate" 2
 		"shield generation" 2
 		"scram drive" 0.4
-		"jump fuel" 100
+		"jump fuel" 75
 		"hyperdrive" 1
```

Comment:
Going to reiterate my concerns that this has `hyperdrive` built in.
It costs 100 fuel to use.
But you're removing 25 fuel, so the Embersylph will never be able to jump. Same with the Embershade.